base/components/Misc.jsx

```diff
@@ -460,7 +460,8 @@ Misc.ImgThumbnail = ({url, alt, background, style, className = ''}) => {
 	// add in base (NB this works with style=null)
 	style = Object.assign({width: '100px', height: '100px', objectFit: 'contain', padding: 0}, style);
 	if (background) style.background = background;
-	if (!alt || !alt.length) {
+	const suppressAccWarnings = DataStore.getUrlValue("suppressAccWarnings");
```

Comment:
Adding a "be quiet" switch makes sense. In fact, let's go further and make it the other way round: Let's only report missing alt tags if a url flag like `accessible=true` is set.